static/app/components/questionTooltip.tsx

```diff
@@ -16,7 +16,7 @@ const QuestionIconContainer = styled('span')<ContainerProps>`
 
   & svg {
     transition: 120ms opacity;
-    color: ${p => p.theme.gray200};
+    color: ${p => p.theme.gray300};
```

Comment:
let's use the alias `subText` instead of `gray300`

Reply:
Good call